### PR TITLE
SQLite & SQLiteDB Helper Tests

### PR DESCRIPTION
**Pull request by rjt-gupta**
Please merge #326 first.

### CODE REVIEW

**Comment by rnehra01** [comment]
Merged.

File: tanner/tests/test_sqlite_db_helper.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            result = self.cursor.fetchall()
+            self.result.append(result)
+            self.handler.delete_db(self.filename)
+
+        self.loop.run_until_complete(test())
+
+        self.assertEqual(self.result, self.expected_result)
+        self.handler.insert_dummy_data.assert_has_calls(calls, any_order=True)
+
+    def test_get_abs_path(self):
```
Comment (rnehra01) [inline tanner/tests/test_sqlite_db_helper.py]: Add an example containing `..`, `/.`

Comment (rjt-gupta) [reply tanner/tests/test_sqlite_db_helper.py]: Added.

File: tanner/tests/test_sqlite_db_helper.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        self.path = 'db/attacker_db'
+        self.returned_result = self.handler.get_abs_path(self.path, '/tmp/')
+        self.expected_result = '/tmp/db/attacker_db'
+        self.assertEqual(self.returned_result, self.expected_result)
+
+    def test_get_abs_path_2(self):
+        self.path = '../tmp/db/test_db'
```
Comment (rnehra01) [inline tanner/tests/test_sqlite_db_helper.py]:
```suggestion
        self.path = '../../tmp/db/./test_db'
```

Comment (rnehra01) [inline tanner/tests/test_sqlite_db_helper.py]:
```suggestion
        self.path = '../../tmp/db/./test_db'
```

**Pull request merged**